src/test/java/de/tum/in/www1/artemis/exercise/programmingexercise/GitServiceTest.java

```diff
@@ -1,25 +1,26 @@
 package de.tum.in.www1.artemis.exercise.programmingexercise;
 
-import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.*;
```

Comment:
what imports to you use besides `assertThat`? Am I overlooking something?

Reply:
assertThatThrownBy but still no star import